requirements-min.txt

```
@@ -1,7 +1,7 @@
 # minimum versions of package dependencies for installing PyNWB
-h5py==3.2.0
+h5py==3.15.1
 hdmf==4.1.1
-numpy==1.24.0
-pandas==1.2.0
+numpy==2.2.6
+pandas==2.3.3
 python-dateutil==2.8.2
-platformdirs==4.1.0
\ No newline at end of file
+platformdirs==4.1.0
```

```suggestion
h5py==3.6.0
hdmf==4.1.1
numpy==1.24.0
pandas==1.3.4
python-dateutil==2.8.2
platformdirs==4.1.0
```

I think we should keep the minimum requirements as low as possible (and this requirements file should match the pyproject.toml dependency constraints). h5py 3.6.0 and pandas 1.3.4 looks like the first to have pre-built packages for python 3.10. I don't think the others should need to be updated.